### PR TITLE
Create jurnal-tsaqafah.csl

### PR DESCRIPTION
**Pull request by daredmayo**

### CODE REVIEW

File: tsaqafah.csl [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+    </author>
+    <contributor>
+      <name>Lalu Ganda</name>
+      <email>laluganda@unida.gontor.ac.id</email>
+    </contributor>
+    <category citation-format="note"/>
+    <category field="theology"/>
+    <issn>1411-0334</issn>
+    <eissn>2460-0008</eissn>
+    <summary>note with bibliography (adapted-modified from jurnal al-jami'ah)</summary>
```
Comment (rmzelle) [inline tsaqafah.csl]: `<summary>note with bibliography (adapted-modified from jurnal al-jami'ah)</summary>`

@daredmayo, does this mean that there is a CSL style for the journal al-jami'ah? Do you know where that style is available? (I don't think we have it in our repository)

File: tsaqafah.csl [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+          </if>
+        </choose>
+        <text macro="journal-page"/>
+        <choose>
+          <if match="none" variable="DOI">
+            <text macro="url" prefix=" "/>
+          </if>
+        </choose>
+      </group>
+      <text variable="DOI" prefix=" [https://doi.org/" suffix="]"/>
```
Comment (rmzelle) [inline tsaqafah.csl]: @daredmayo, I changed the DOI prefix from `http://dx.doi.org/` to https://doi.org/`, which is nowadays the preferred format.

File: tsaqafah.csl (hunk 1 above)
Comment (daredmayo) [reply tsaqafah.csl]: yes, there is CSL styl for journal al-jami'ah...
i found it here http://aljamiah.or.id/support/others/al-jamiah-journal-of-islamic-studies-indonesia.csl

Comment (rmzelle) [reply tsaqafah.csl]: Could you please ask the editor of that journal (at editor@aljamiah.org) whether they are okay with you submitting a modified version here under our required "Creative Commons Attribution-ShareAlike 3.0 License" license?

http://aljamiah.or.id/support/others/al-jamiah-journal-of-islamic-studies-indonesia.csl doesn't have any licensing information, so we rather make sure.

Comment (daredmayo) [reply tsaqafah.csl]: Thank you, i will contact that editor to make sure for the license

Comment (rmzelle) [reply tsaqafah.csl]: By the way, we'd also be happy to add their style to our repository, if they're interested in that.

Comment (daredmayo) [reply tsaqafah.csl]: i have contact Al-Jami'ah editor and they have no problem about the licence

**Comment by rmzelle** [comment]
Thanks!

File: tsaqafah.csl (hunk 1 above)
Comment (rmzelle) [reply tsaqafah.csl]: Great, thanks for checking.

**Pull request merged**